api/v1alpha1/progressiverollout_types.go

```diff
@@ -90,6 +90,16 @@ func (in *ProgressiveRollout) NewStatusCondition(t string, s metav1.ConditionSta
 	}
 }
 
+// HasOwnerReference returns true if the ProgressiveRollout object has a reference to one of the owners
+func (in *ProgressiveRollout) HasOwnerReference(owners []metav1.OwnerReference) bool {
```

Comment:
Should this be some thing like `IsOwnedBy`? I initially read it as if the `ProgressiveRollout` had any owner reference on it.

Reply:
Fixed